articles/index.yml

```
@@ -31,6 +31,8 @@ landingContent:
             url: overview/what-is-qsharp-and-qdk.md
           - text: Linear algebra for quantum computing
             url: overview/algebra-for-quantum-computing.md
+          - text: Microsoft Learn modules for quantum computing
+            url: https://docs.microsoft.com/learn/browse/?term=quantum
```

```suggestion
            url: https://docs.microsoft.com/en-us/users/buildcollections2020-6557/collections/1o2iogrmn8x4r
```
This is the link we use at Build.